src/components/dashboard/ClaimButton.js

```diff
@@ -74,7 +74,6 @@ const getStylesFromProps = ({ theme }) => ({
   amountInButton: {
     display: 'inline',
     marginLeft: theme.sizes.defaultHalf,
-    lineHeight: theme.sizes.default * 3,
```

Comment:
without it, the `1 G$` was rendered below the line of the text (chrome responsive/iphone5 simulated view).

but if you happen to see it right, then I can consider it an issue on my end.

Reply:
please test on your end again what happens if you remove the lineheight. send an image if it doesnt look good

Reply:
on responsive view, it's ok. For the iPhone 5/SE simulation, this is the result

![line-height](https://user-images.githubusercontent.com/3315606/64434452-f7a57b80-d096-11e9-9258-4b2dfadfcd4a.gif)

Reply:
i've added lineheight to the inner items